deps/compiler-rt_standalone.mk

```diff
@@ -0,0 +1,90 @@
+##
+# This Makefile will be executed in $(BUILDDIR)/compiler-rt-$(LLVM_VER)
+# Expected variables from the parent
+# CRT_SRCDIR
+# LIBFILE
+# SLIBFILE
+# OS (CRT_OS not JL_OS)
+# ARCH (CRT_ARCH not JL_ARCH)
+# USE_CLANG
+# fPIC
+##
+
+# The standalone compiler-rt build is inspired by
+# https://github.com/ReservedField/arm-compiler-rt
+SRCDIR := $(CRT_SRCDIR)/lib/builtins
+
+ifeq ($(ARCH), armhf)
+ARCH_SRCDIR := $(SRCDIR)/arm
+else ifeq ($(ARCH), aarch64)
+ARCH_SRCDIR := $(SRCDIR)/arm64
+else
+ARCH_SRCDIR := $(SRCDIR)/$(ARCH)
+endif
+
+INCLUDES := -I$(SRCDIR) -I$(ARCH_SRCDIR)
+# TODO(vchuravy) discover architecture flags
+CRT_CFLAGS := $(CPPFLAGS) $(CFLAGS) -O2 -std=c11 \
+		$(fPIC) $(INCLUDES) \
+		-fno-builtin -ffreestanding
+ifeq ($(USE_CLANG),1)
+CRT_CFLAGS += -Wno-unknown-attributes -Wno-macro-redefined
+endif
+
+##
+# Blacklist a few files we don't want to deal with
+##
+MAKEFLAGS := --no-builtin-rules
+BLACKLIST := atomic.o atomic_flag_clear.o atomic_flag_clear_explicit.o \
```

Comment:
@tkelman the `emutls` failure means we need to update the blacklist to contain the related files